tools/rosbag/src/record.cpp

```diff
@@ -299,6 +314,8 @@ int main(int argc, char** argv) {
     // Run the recorder
     rosbag::Recorder recorder(opts);
     int result = recorder.run();
-    
+    ros::waitForShutdown();
+
     return result;
 }
+
```

Comment:
Nitpick: please revert unrelated white space changes like this.

Reply:
Done